javalib/src/main/scala/java/util/BitSet.scala

```diff
@@ -0,0 +1,678 @@
+package java.util
+
+// Ported from Scala.js
```

Comment:
Do you mind putting the SHA and the date of the port? There are some other files in the javalib you can use as an example, maybe HashMap? Also for the test file.

Reply:
Yes, absolutely! Done.